libs/core/tests/unit_tests/runnables/test_runnable.py

```diff
@@ -4014,7 +4014,7 @@ def test_runnable_lambda_stream_with_callbacks() -> None:
     llm = FakeStreamingListLLM(responses=[llm_res], sleep=0.01)
     config: RunnableConfig = {"callbacks": [tracer]}
 
-    assert list(RunnableLambda(lambda _: llm).stream("", config=config)) == list(
+    assert list(RunnableLambda(lambda _: llm).stream("", config=config)) == list(  # type: ignore[comparison-overlap]
```

Comment:
I couldn't make typing work correctly when the input callable returns a `Runnable`